README.rst

```diff
@@ -3,14 +3,16 @@ The Ubuntu Packaging Guide
 
 This guide is often one of the first things that a new Ubuntu contributor will
 look at. Helping make it better can have a big impact.
-
 The actual articles can be found under the ``docs/`` directory in rst files.
 Images can be placed in the ``docs/images/`` folder.
+The html, js and css files can be found under ``docs/_static``. 
```

Comment:
suggestion: If you still want to land these changes we could
1. either `git rebase` this branch and only pick this change. Feel free to reach out to me if you need help/guidance with that
2. or close this PR and you open another PR with this change.

Reply:
note: I will close the PR for now, but if you want to pursue option 1 than I am happy to reopen it.